src/Symfony/Bundle/FrameworkBundle/Tests/Functional/CachePoolsTest.php

```diff
@@ -50,7 +60,7 @@ public function testRedisCustomCachePools()
         try {
             $this->doTestCachePools(array('root_config' => 'redis_custom_config.yml', 'environment' => 'custom_redis_cache'), RedisAdapter::class);
         } catch (\PHPUnit_Framework_Error_Warning $e) {
-            if (0 !== strpos($e->getMessage(), 'unable to connect to 127.0.0.1')) {
+            if (0 !== strpos($e->getMessage(), 'unable to connect to')) {
```

Comment:
Both return false and behaviour will be the same (exception never thrown). Note that this doesn't apply to the previous test, where exception is indeed thrown and the test is skipped properly.